longevity_test.py

```diff
@@ -42,7 +42,7 @@ def _get_tombstone_gc_verification_params(self) -> dict:
         return params
 
     def _get_scan_operation_params(self) -> list[FullScanParams]:
-        params = self.params.get("run_fullscan", {})
+        params = self.params.get("run_fullscan", [])
```

Comment:
i think it should not work, since #2823

Reply:
Guess which commit brought it back

https://github.com/scylladb/scylla-cluster-tests/blame/73dc21fe6b2fdd387986f9515bcdca9945002bc8/sdcm/sct_config.py#L1829

Reply:
@temichus , please revert this change (from the commit above) and use as we currently do in the entire SCT

Reply:
i always forget about this was removed from `self.params`